lms/envs/production.py

```diff
@@ -1044,6 +1044,7 @@ def should_show_debug_toolbar(request):
 ZOHO_REFRESH_ENDPOINT = AUTH_TOKENS.get('ZOHO_REFRESH_ENDPOINT')
 ZOHO_STUDENTS_ENDPOINT = AUTH_TOKENS.get('ZOHO_STUDENTS_ENDPOINT')
 ZOHO_MENTORS_ENDPOINT = AUTH_TOKENS.get('ZOHO_MENTORS_ENDPOINT')
+ZOHO_TIMEOUT_SECONDS = float(AUTH_TOKENS.get('ZOHO_TIMEOUT_SECONDS', 0.5))
```

Comment:
Looks good, just two questions:
- Is there a particular reason to use `AUTH_TOKENS` here as opposed to the more general `ENV_TOKENS`?
- Aren't both these `_TOKENS` variables json that's already been parsed? If so, we shouldn't need an extra `float()` here, right? I see that lines above such as `ZOHO_REFRESH_SLEEP_SECS` don't do any casting.

Reply:
good point, will remove the parse 

in Juniper, the env_tokens and auth_tokens have been combined.

Reply:
Ah, cool, didn't know that.
I did see a ton of `ENV_TOKENS` elsewhere in that file - https://github.com/Code-Institute-Org/edx-juniper/blob/796755f0dc98926d7b6ad48c2f4c42cc24833e6d/lms/envs/production.py

But you're saying that we're now only using `AUTH_TOKENS` for our stuff? If so, that's cool, simpler is better :)